l10n_ar_account_tax_settlement/models/account_tax.py

```diff
@@ -0,0 +1,8 @@
+from odoo import api, models, fields
```

Comment:
remove api import, not used
```suggestion
from odoo import models, fields
```